1.9/administration/installing/custom/configuration-parameters.md

```diff
@@ -20,6 +20,36 @@ This parameter specifies a YAML nested list (`-`) of IPv4 addresses to your [pri
 ### bootstrap_url
 This required parameter specifies the URI path for the DC/OS installer to store the customized DC/OS build files. If you are using the automated DC/OS installer, you should specify `bootstrap_url: file:///opt/dcos_install_tmp` unless you have moved the installer assets. By default the automated DC/OS installer places the build files in `file:///opt/dcos_install_tmp`.
 
+### cluster_docker_credentials
+This parameter specifies a dictionary of Docker credentials to pass. 
+
+- If unset, the default is an empty credentials file is placed in the
+`/etc/mesosphere/docker_credentials` during DC/OS install. You can then use sysadmin scripts to edit the file and do a `systemctl restart dcos-mesos-slave` or `systemctl restart dcos-mesos-slave-public`.
+- You can also set as the `--docker_config` JSON [format](http://mesos.apache.org/documentation/latest/configuration/). In the `config.yaml` file you can write YAML and it will automatically be mapped to the JSON format for you. This will store the Docker credentials in the same location as the DC/OS internal configuration (`/opt/mesosphere`). If you need to update or change the configuration, you will have to create a new DC/OS internal configuration.
```

Comment:
`You can also specify by using the `--docker_config` JSON [format](http://mesos.apache.org/documentation/latest/configuration/). `